docs/guide/design-principles/minimize-coordination-content.md

```diff
@@ -30,7 +34,7 @@ You can use a pattern such as [Scheduler Agent Supervisor][sas-pattern] to coord
 
 These two patterns complement each other. If the write-only store in CQRS uses event sourcing, the read-only store can listen for the same events to create a readable snapshot of the current state, optimized for queries. Before adopting CQRS or event sourcing, however, be aware of the challenges of this approach.
 
-**Partition data.** Avoid putting all of your data into one data schema that is shared across many application services. A microservices architecture enforces this principle by making each service responsible for its own data store. Within a single database, partitioning the data into shards can improve concurrency, because a service writing to one shard does not affect a service writing to a different shard.
+**Partition data and state.** Even though partitioning adds some degree of coordination, you can use partitioning to leverage parallelism for better scalability. You can partition your monolithic state into smaller chunks which are managed independently from each other. This helps with scalability and fault tolerance but at the cost of consistency and simplicity. You can also partition data to improve concurrency. Avoid putting all of your data into one data schema that is shared across many application services. A microservices architecture enforces this principle by making each service responsible for its own data store. Within a single database, partitioning the data into shards can improve concurrency, because a service writing to one shard does not affect a service writing to a different shard.
```

Comment:
```suggestion
**Partition data and state.** Avoid putting all of your data into one data schema that is shared across many application services. A microservices architecture enforces this principle by making each service responsible for its own data store. Within a single database, partitioning the data into shards can improve concurrency, because a service writing to one shard does not affect a service writing to a different shard. Even though partitioning adds some degree of coordination, you can use partitioning to increase parallelism for better scalability. Partition monolithic state into smaller chunks so the data can be managed independently.
```